easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PerlModule'
+
+name = 'Text-CSV'
+version = '1.33'
+#versionsuffix = '-Perl-%(perlver)s-bare' 
```

Comment:
@RvDijk why comment this out, it's still relevant even if the Perl dep is a bare variant?